main.go

```diff
@@ -136,6 +138,8 @@ func main() {
 	flags.StringVar(&opt.GoogleServiceAccountCredentialFile, "google-service-account-credential-file", os.Getenv("GOOGLE_APPLICATION_CREDENTIALS"), "location of a credential file described by https://cloud.google.com/docs/authentication/production")
 	flags.StringVar(&opt.GoogleOAuthClientCredentialFile, "google-oauth-credential-file", opt.GoogleOAuthClientCredentialFile, "location of a credential file described by https://developers.google.com/people/quickstart/go, setup from https://cloud.google.com/bigquery/docs/authentication/end-user-installed#client-credentials")
 
+	flags.StringVar(&opt.PinnedDateTime, "pinnedDateTime", opt.PinnedDateTime, "optional value to use instead of the current time specified in RFC3339 format - 2006-01-02 15:04:05+00:00")
```

Comment:
Let's add a note that this is for historical sippy instances.

Reply:
However do we actually need a CLI option? My thought here is this value can be read from the db, find the last prow job run timestamp.